docusaurus/docs/develop/path/cheatsheet.md

```diff
@@ -0,0 +1,240 @@
+
+If you do not wish to use an OAuth provider, simply answer `no` when prompted. This will allow authorizing requests with a static API key only.
+
+:::
+
+<!-- TODO_TECHDEBT(@commoddity): Update the `make init_envoy` script to remove the JWT HTTP filter if the operator selects to use a static API key only. -->
+
+## 3. Run the `PATH` Gateway
+
+```bash
+make path_up
+```
+
+This will start the `PATH` Gateway.
+
+You should see the following output:
+
+```bash
+❯ make path_up
+#########################################################################
+### ./local/path/config/.config.yaml already exists, not overwriting. ###
+#########################################################################
+No kind clusters found.
+Cluster 'path-localnet' not found. Creating it...
+Creating cluster "path-localnet" ...
+ ✓ Ensuring node image (kindest/node:v1.31.2) 🖼
+ ✓ Preparing nodes 📦
+ ✓ Writing configuration 📜
+ ✓ Starting control-plane 🕹️
+ ✓ Installing CNI 🔌
+ ✓ Installing StorageClass 💾
+Set kubectl context to "kind-path-localnet"
+You can now use your cluster with:
+
+kubectl cluster-info --context kind-path-localnet
+
+Thanks for using kind! 😊
+Switched to context "kind-path-localnet".
+Checking if secret 'path-config-local' exists...
+Secret 'path-config-local' not found. Creating it...
+secret/path-config-local created
+Tilt started on http://localhost:10350/
+v0.33.21, built 2024-11-08
+
+(space) to open the browser
+(s) to stream logs (--stream=true)
+(t) to open legacy terminal mode (--legacy=true)
+(ctrl-c) to exit
+```
+
+You can visit http://localhost:10350 in your browser to view the Tilt dashboard, which allows you to view the log output for all running containers.
+
+:::caution
+
+The `PATH Gateway` stack may take a minute or more to initialize the first time you run it as it must download all required Docker images.
+
+You will be able to tell it is ready when you see log output like this in the `path` Resource in the Tilt dashboard:
+
+```json
+{"level":"info","message":"Starting PATH gateway with Shannon protocol"}
+{"level":"info","message":"Starting the cache update process."}
+{"level":"info","package":"router","message":"PATH gateway running on port 3000"}
+{"level":"info","services count":1,"message":"Running Hydrator"}
+```
+
+:::
+
+Once the `PATH Gateway` container is ready, you may send a relay to test.
+
+## 4. Send a Relay
+
+Check that the `PATH Gateway` is serving relays by running the following command yourself:
+
+```bash
+curl http://eth.localhost:3001/v1/endpoint_1 \
+    -X POST \
+    -H "Content-Type: application/json" \
+    -H "Authorization: api_key_1" \
+    -d '{"jsonrpc": "2.0", "id": 1, "method": "eth_blockNumber" }'
+```
+
+:::tip
+
+The `GatewayEndpoint` ID `endpoint_1` and API Key `api_key_1` are defined in the `local/path/envoy/.gateway-endpoints.yaml` file.
```

Comment:
Can we rename `endpoint_XXX` to be more semantic?

Left similar comments elsewhere

Reply:
Done.